### PR TITLE
Dev/GitHub#69 daten inaktivierung

### PR DESCRIPTION
**Pull request by AlwinEgger**

### CODE REVIEW

File: src/main/resources/application.conf [hunk 1]
```diff
@@ -269,16 +269,16 @@ openolitor {
     # Project specific akka persistence configuration
     akka-persistence-sql-async {
       url = "jdbc:mysql://localhost:3306/csa1"
-      user = "dbusercsa1"
-      password = "thisDefaultUserPasswordMustBeChanedInProduction"
+      user = "root"
+      password = "thedefaultdbrootpasswordneedstobechanged"
```
Comment (mcmpp) [inline src/main/resources/application.conf]: do you need to replace this?? I use this one in my local env

File: src/main/resources/application.conf [hunk 2]
```diff
@@ -269,16 +269,16 @@ openolitor {
...
     }
 
     # Mandant specific db settings
     db: {
       default: {
         url = "jdbc:mysql://localhost:3306/csa1"
-        user = "dbusercsa1"
-        password = "thisDefaultUserPasswordMustBeChanedInProduction"
+        user = "root"
```
Comment (mcmpp) [inline src/main/resources/application.conf]: do you need to replace this?? I use this one in my local env

File: src/main/resources/logback.xml [hunk 3]
```diff
@@ -57,6 +57,11 @@
         <appender-ref ref="STDOUT"/>
     </logger>
 
+    <!-- TO SEE SQL QUERY LOG-OUTPUT SWITCH THIS LEVEL TO DEBUG-->
+    <logger name="scalikejdbc" level="INFO" additivity="false">
```
Comment (mcmpp) [inline src/main/resources/logback.xml:61]: not too much printing??

**Review by mcmpp — APPROVED** [review]

File: src/main/resources/application.conf (hunk 2 above)
Comment (AlwinEgger) [reply src/main/resources/application.conf]: no, this was a mistake! I revert it.

File: src/main/resources/application.conf (hunk 1 above)
Comment (AlwinEgger) [reply src/main/resources/application.conf]: no, this was a mistake! I revert it.

File: src/main/resources/logback.xml (hunk 3 above)
Comment (AlwinEgger) [reply src/main/resources/logback.xml:61]: No. Root was at INFO, too. So no change.
 If you change it (locally) to DEBUG, you get the SQL-statements.

**Pull request merged**